Files/ViewModels/NavToolbarViewModel.cs

```diff
@@ -604,9 +606,36 @@ public void VisiblePath_QuerySubmitted(AutoSuggestBox sender, AutoSuggestBoxQuer
             (this as INavigationToolbar).IsEditModeEnabled = false;
         }
 
-        public void PathBoxItem_Tapped(object sender, TappedRoutedEventArgs e)
+        public async Task PathBoxItem_PointerPressed(object sender, PointerRoutedEventArgs e)
```

Comment:
Any reason for this to be an async Task method? IMO a public void would be enough.

Reply:
I was following best practice (and Roslyn analyser recommendations) - but you are right. It isn't being called by an async method, so either way it doesn't make any difference.

Reply:
Ok, then:

```suggestion
        public void PathBoxItem_PointerPressed(object sender, PointerRoutedEventArgs e)
```